adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/ClientCreds/JsonWebToken.cs

```diff
@@ -91,10 +91,10 @@ public JsonWebToken(IClientAssertionCertificate certificate, string audience)
             };
         }
 
-        public ClientAssertion Sign(IClientAssertionCertificate credential)
+        public ClientAssertion Sign(IClientAssertionCertificate credential, bool sendX5C)
```

Comment:
Why this parameter called sendX5C ? 
this Method create ClientAssertion object from this (JsonWebToken) object, there is no such thing as send (this class should not know about how it will be used in outer context)

Reply:
also X5C is not precise enough - is it thumbprint or cert value or both

Reply:
@trwalke

---
In reply to: [202845504](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1124#discussion_r202845504) [](ancestors = 202845504)

Reply:
Updated parameter to "sendCertificate"